.appveyor.yml

```diff
@@ -50,18 +63,21 @@ test_script:
       "--------------------------------------------"
   - cmd: npm run %vscode%
   - ps: |
-      npm run coverage:system-tests
       npm run aggregateJUnit
       junit-merge -d junit-aggregate -o junit-aggregate.xml
       $NpmTestsExitCode = $LastExitCode
       # Show status
       "`n`n"
       "Finalizing results`n"
-      $AllFiles = Get-ChildItem -Path $ProjectRoot\packages\*\xunit.xml -File | Select -ExpandProperty FullName
+      $AllFiles = Get-ChildItem -Path $ProjectRoot\packages\*\xunit*.xml -File | Select -ExpandProperty FullName
```

Comment:
Modified path to check for the new xunit variations (unit, integration, vscodeIntegration).